dandi/support/pyout.py

```diff
@@ -36,7 +36,7 @@ def naturalsize(v):
     return humanize.naturalsize(v)
 
 
-def datefmt(v, fmt=u"%Y-%m-%d/%H:%M:%S"):
+def datefmt(v, fmt="%Y-%m-%d/%H:%M:%S"):
```

Comment:
pre-commit will not let me restore these changes - I tried manually putting them back to the way they were but every time I try to commit it, pre-commit triggers and undoes them.

Reply:
I can certainly try it, but you can always squash it on merge to the same effect, though, right?

*E.g.*,

![image](https://user-images.githubusercontent.com/51133164/188206548-7fdce7c8-0b09-44c5-9852-f2d792ca0a3d.png)

A bit less work that way IMO

Reply:
this PR still has unrelated changes we do not want to merge/squash in, right?

Reply:
Not since I forcibly restored the changes to `supper/pyout.py`. Now the changelog represents only the related changes to the NWB Inspector and NWB asset validation.

Just wanted you to be aware that following [jwodder's instructions](https://github.com/dandi/dandi-cli/pull/1108#discussion_r959937152) to the letter resulted in pre-commit automatically doing all of the changes to that file.

Reply:
dandi/tests/fixtures.py is still in the diff and to be gone if you like to continue your fight!

Reply:
The changes to `dandi/tests/fixtures.py` are intentional updates to keep them compatible with the new validation requirements

Reply:
The git stash step didn't work out for me, but

```
git reset $(git merge-base master $(git branch --show-current))
git add -A
git commit --no-verify -m "manual squash"  # Need --no-verify to skip the pyout.py changes
git push -f
```

did